infra/gp-identity-provider/helper/generate_encrypted_value.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+filename="encryptMe.yml"
+
+if [ ! -e ${filename} ];
+then
+  echo "ERROR: Es gibt keine Datei ${filename}"
+  exit 1
+fi
+
+echo -e "The encrypted Value: \n"
+oc create secret generic encryptedValue --from-file=encryptedValue.yaml=${filename} --dry-run=client -o yaml -n openshift-config | kubeseal -o yaml --cert <(oc -n gp-infrastructure get secret sealed-secret-keys -o jsonpath="{.data.tls\.crt}" | base64 -d) | grep encryptedValue.yaml | tr -s " " | cut -d " " -f3
```

Comment:
nach dem `| kubeseal` gehört auch der TargetNamespace nochmal angegeben. Ansonsten nimmt er den Namespace aus dem Current-Context deiner kubeconfig. Wenn die auf openshift-config steht, passts, wenn nicht kannst du das secret nicht decrypten.

Reply:
-n beim erstellen des secrets ist scheinbar genug, kubseal orientiert sich am namspace des secrets.(?)

Habs kurz ausprobiert: 

namespace im secret -> openshift-config
namespace im context -> default

konnte mir das secret in der web console, im namespace openshift-config, decrypted anzeigen lassen.

falls hier was zu machen ist zieh ichs im original script auch nach, war mostly copy/paste.